packages/plugins/rpc-manager/src/lib/eth_signData.ts

```diff
@@ -55,7 +52,7 @@ export default class EthSignData extends RpcModifier {
       this.logger.trace(__(`Modifying blockchain '${params.request.method}' response:`));
       this.logger.trace(__(`Original request/response data: ${JSON.stringify(params)}`));
 
-      const account = accounts.find(acc => (
+      const account = this.accounts.find(acc => (
         Web3.utils.toChecksumAddress(acc.address) ===
```

Comment:
Should it be `acc.address || acc` as seen in [other lines](https://github.com/embark-framework/embark/pull/2150/files#diff-7f2d476e5b66f22cdc294475317d84f9R33)?

Reply:
Have removed `acc.address || acc` from the other source as it was not needed.